caffe2/python/serialized_test/SerializedTestCoverage.md

```diff
@@ -0,0 +1,725 @@
+# Serialized Test Coverage Report
```

Comment:
How do you plan to keep this file up to date? Since it is automatically generated, it will be very easy for people to not update it when they increase coverage. One simple way to test for this is to add something to CI which fails if the checked in file is not up-to-date. Another way is to just not check in the generated file.

Reply:
The file is automatically generated whenever someone generates a new serialized test (logic lives in serialized_test_util.py).